_data/banking.yml

```diff
@@ -66,12 +66,12 @@ websites:
       img: andrewsfederalcu.png
       tfa: No
 
-    - name: Australia and New Zealand Banking Group
+    - name: Australia and New Zealand Banking Group (ANZ)
       url: https://www.anz.com.au/
-      twitter: ANZ_AU
-      facebook: ANZAustralia
       img: anz.png
-      tfa: No
+      tfa: Yes
```

Comment:
I don't think this is correct. on the `doc` URL it states

> Once you've registered for ANZ Shield, you'll be prompted to generate a Shield Code when making internet banking transactions or any changes to your account.

So this is two-factor _authorization_, not authentication..